analyticConfigurations/computed_metric-elastic.search.json

```diff
@@ -4,239 +4,300 @@
       {
         "match": "netuitive.linux.elasticsearch.cpu.normalizedpercent",
         "properties": {
-          "expression": "attribute['cpus'] == null ? null : (data['elasticsearch.process.cpu.percent'].actual / attribute['cpus'].value)",
+          "expressions": [
+            "${elasticsearch.process.cpu.percent}.actual / attribute(cpus)",
+            "0"
+          ],
           "fqn": "netuitive.linux.elasticsearch.cpu.normalizedpercent"
         }
       },
       {
         "match": "elasticsearch.indices.index.flush.avg_time_in_millis",
         "properties": {
-          "expression": "data['elasticsearch.indices.${1}.flush.total_time_in_millis'].actual / data['elasticsearch.indices.${1}.flush.total'].actual",
-          "for": "elasticsearch\\.indices\\.(.*?)\\.flush\\.total_time_in_millis",
+          "expressions": [
+            "${elasticsearch.indices.${1}.flush.total_time_in_millis}.actual / ${elasticsearch.indices.${1}.flush.total}.actual"
+          ],
+          "capture": "elasticsearch\\.indices\\.(.*?)\\.flush\\.total_time_in_millis",
```

Comment:
@jemassey these captures don't seem to be capturing any metrics. Do you see anything obviously wrong with them?